lib/src/model/product_image.dart

```diff
@@ -179,5 +187,36 @@ class ProductImage {
       '${y1 == null ? '' : ',y1=$y1'}'
       '${x2 == null ? '' : ',x2=$x2'}'
       '${y2 == null ? '' : ',y2=$y2'}'
+      '${width == null ? '' : ',width=$width'}'
+      '${height == null ? '' : ',height=$height'}'
       ')';
+
+  @override
+  int get hashCode =>
+      '${field.offTag}_${language?.code}_${size?.offTag}'.hashCode;
```

Comment:
Can't we just use the toString method here

Reply:
In that case we could probably, but I don't think a hashcode should depend on other unrelated methods. What if the implementation of `toString` is removed from that class?

Fun facts:
1. I don't care about `hashcode`, it's just that the operator `==` triggers a warning about un-implemented `hashcode` method
2. with your logic I could have implemented `==` with `toString` too. That would have been ugly